packages/runtime/container-runtime/src/dataStores.ts

```diff
@@ -363,13 +518,8 @@ export class DataStores implements IDisposable {
 
 	/** Package up the context's attach summary etc into an IAttachMessage */
 	private generateAttachMessage(localContext: LocalFluidDataStoreContext): IAttachMessage {
-		const { attachSummary, type, gcData } = localContext.getAttachData(
-			/* includeGCData: */ true,
-		);
-
-		if (gcData !== undefined) {
-			addBlobToSummary(attachSummary, gcDataBlobKey, JSON.stringify(gcData));
-		}
+		const { attachSummary } = localContext.getAttachData(/* includeGCData: */ true);
```

Comment:
Why move the GC data part into the data store context?
@markfields FYI since you recently added this.